### ISSUE TITLE
NPC 'Stillpine Ancestor Akida' for quest 'Totem of Coo' runs too fast

### ISSUE THREAD

**Issue by MsgLosers** [dalaranwow/dalaran-wow#5673]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
The NPC will run ahead of you to show you the path, but he is running at like 150 % the normal running speed, so he will outrun you, leaving you behind (especially if you end up in a battle, which you will because the path contains unavoidable mobs).

**Expected behaviour**: Tell us what should happen instead
Make him run at 100 or (better) 90 % the normal running speed so you can keep up and catch up to him.

**Steps to reproduce the problem**:

1. Start level 8 quest "Totem of Coo"
2. NPC will automatically spawn and tell you to follow.
3. He outruns you at 150 % speed so you can never catch up.

**Include proofs for this behaviour**

[](https://i.paste.pics/166049a9c00dee113bf3897d92a7b1ed.png "Proof")
https://i.paste.pics/166049a9c00dee113bf3897d92a7b1ed.png

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**
